sip-servlets-test-suite/applications/quoted-parameter-sip-servlet/pom.xml

```diff
@@ -0,0 +1,24 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
```

Comment:
do we need a full new app just to test the param features. Couldnt we reuse an existing app, and add test-case branch...??